packages/platform-android/src/link/warnAboutManuallyLinkedLibs.ts

```diff
@@ -1,18 +1,16 @@
-// @flow
-
 import chalk from 'chalk';
 import {logger} from '@react-native-community/cli-tools';
-import type {ConfigT} from 'types';
 import getLinkConfig from './index';
+import {Config} from '../types';
 
 // TODO: move to cli-tools once platform-ios and platform-android are migrated
 // to TS and unify with iOS implementation
 export default function warnAboutManuallyLinkedLibs(
-  config: ConfigT,
+  config: Config,
   platform: string = 'android',
-  linkConfig: $Call<typeof getLinkConfig> = getLinkConfig(),
+  linkConfig: any = getLinkConfig(),
```

Comment:
here's the type for that: https://github.com/react-native-community/cli/blob/eddfd4d20a62d11133687843b52f7095c5c88118/types/index.js#L83-L93

And you already have it here: https://github.com/react-native-community/cli/pull/597/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts#diff-0e7f694a165ac213be4e26f76b7179d3R42